fuzz/fuzz_targets/streams.rs

```diff
@@ -49,16 +49,22 @@ fuzz_target!(|input: (StreamParams, Vec<Operation>)| {
                 stream.accept(dir);
             }
             Operation::Finish(id) => {
-                stream.finish(id);
+                if let Ok(x) = stream.finish(id) {
+                    x
+                }
             }
             Operation::ReceivedStopSending(sid, err_code) => {
                 stream.received_stop_sending(sid, err_code);
             }
             Operation::ReceivedReset(rs) => {
-                stream.received_reset(rs);
+                if let Ok(x) = stream.received_reset(rs) {
+                    assert!(x & true | true);
+                }
             }
             Operation::Reset(id) => {
-                stream.reset(id);
+                if let Ok(x) = stream.reset(id) {
+                    x
+                }
```

Comment:
What do the above three changes accomplish? They look like no-ops to me.

Reply:
That's correct. This change and those below are lint fixes raised from applying clippy to the fuzz package. These changes fix warning on unused `Result`. I'd opted not to ignore any lints that weren't already ignored in quinn but I'd be happy to revert the changes in this file and instead add an allow for the lint warning.

Reply:
Rather than blanket suppressing lints, let's explicitly discard results we don't care about with `let _ = ...`. As is, this code is confusing.

Reply:
yep I agree, sounds good